databricks/sdk/mixins/jobs.py

```diff
@@ -0,0 +1,30 @@
+from databricks.sdk.service import jobs
+
+
+class JobsExt(jobs.JobsAPI):
+
+    def get_run(self,
+                run_id: int,
+                *,
+                include_history: bool | None = None,
+                include_resolved_values: bool | None = None,
+                page_token: str | None = None) -> jobs.Run:
+        run = super().get_run(run_id,
+                              include_history=include_history,
+                              include_resolved_values=include_resolved_values,
+                              page_token=page_token)
+
+        isPaginatingIterations = run.iterations is not None and len(run.iterations) > 0
+
+        while run.next_page_token:
+            next_run = super().get_run(run_id,
+                                       include_history=include_history,
+                                       include_resolved_values=include_resolved_values,
+                                       page_token=run.next_page_token)
+            if (isPaginatingIterations):
+                run.iterations.extend(next_run.iterations)
+            else:
+                run.tasks.extend(next_run.tasks)
+            run.next_page_token = next_run.next_page_token
+
+        return run
```

Comment:
Something I'm proposing on the Java implementation is to remove any pagination tokens on the response to avoid confusion in this temporary implementation: https://github.com/databricks/databricks-sdk-java/pull/324/files#diff-f9da6e7e4f592f5182710ff4932320564d963bfebc7b27ea075f97d9b8788048R45-R47

It is optional and opinionated, but I guess it would make sense to at least be consistent about it across the different SDKs. Let me know if you disagree.